bundle/deploy/terraform/destroy.go

```diff
@@ -66,7 +17,6 @@ func (w *destroy) Name() string {
 func (w *destroy) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagnostics {
 	// return early if plan is empty
 	if b.Plan.IsEmpty {
-		cmdio.LogString(ctx, "No resources to destroy in plan. Skipping destroy!")
```

Comment:
Could you still log a debug message here?

Reply:
Done